lib/kino/proxy.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Kino.Proxy do
+  @moduledoc """
+  A kino for handling proxy requests from the host.
```

Comment:
Let's not use the word kino, since it basically means an output :)

```suggestion
  Functionality for handling proxy requests forwarded from Livebook.
```

Reply:
```suggestion
  Handle proxy requests from Livebook.
```

We will need to add extensive documentation on the URL format and include examples. :)